source/transactions/tests/create-collection.yml

```diff
@@ -0,0 +1,155 @@
+runOn:
+  -
+    minServerVersion: "4.3.4"
+    topology: ["replicaset", "sharded"]
+
+database_name: &database_name "transaction-tests"
+collection_name: &collection_name "test"
+
+data: []
+
+tests:
+  - description: explicitly create collection using create command
+
+    operations:
+      - name: dropCollection
+        object: database
+        arguments:
+          collection: *collection_name
+      - name: startTransaction
+        object: session0
+      - name: createCollection
+        object: database
+        arguments:
+          session: session0
+          collection: *collection_name
+      - name: assertCollectionNotExists
+        object: testRunner
+        arguments:
+          database: *database_name
+          collection: *collection_name
+      - name: commitTransaction
+        object: session0
+      - name: assertCollectionExists
+        object: testRunner
+        arguments:
+          database: *database_name
+          collection: *collection_name
+
+    expectations:
+      - command_started_event:
+          command:
+            drop: *collection_name
+            writeConcern:
+          command_name: drop
+          database_name: *database_name
+      - command_started_event:
+          command:
+            create: *collection_name
+            lsid: session0
+            txnNumber:
+              $numberLong: "1"
+            startTransaction: true
+            autocommit: false
+            writeConcern:
+          command_name: create
+          database_name: *database_name
+      - command_started_event:
```

Comment:
It seems odd to be asserting an expectation due to a command that is run solely to assert something in the test harness.  Perhaps better to use a separate MongoClient to run listCollections and listIndexes, and that way they won't show up in command monitoring events.  I think there is already such a MongoClient lying around that is used for asserting the "outcome" of the operations

Reply:
Good point. I've removed the assertions.